src/js/media/html5.js

```diff
@@ -74,8 +75,15 @@ vjs.Html5.prototype.createEl = function(){
       player.tag = null;
     } else {
       el = vjs.createEl('video');
+
+      // determine if native controls should be used
+      attributes = videojs.util.mergeOptions({}, player.tagAttributes);
+      if (!vjs.TOUCH_ENABLED || player.options()['nativeControlsForTouch'] === false) {
```

Comment:
The default here changed. See [line 33](https://github.com/videojs/video.js/pull/1811/files#diff-97b7590f7a7a76072c2d0111120738e7R33). (an example of why it'd be good to have things in the same place if we can figure it out some how)